eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -0,0 +1,388 @@
+"""
+Integration test suite for the API v1 views.
+"""
+
+from __future__ import annotations
+
+import requests
+from ddt import data as ddt_data
+from ddt import ddt, unpack
+from django.http import HttpResponse
+from django.test import TestCase
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+
+fake = Faker()
+
+
+API_TIMEOUT = 5
+CLIENT_ID = "client_id"
+CLIENT_SECRET = "client_secret"
+FAKE_USERS_DATA = iter(
+    [
+        {
+            "username": fake.unique.user_name(),
+            "email": fake.unique.email(),
+            "fullname": fake.unique.name(),
+            "password": fake.unique.password(),
+            "activate_user": True,
+        }
+        for _ in range(100)
+    ]
+)
```

Comment:
1. We could generate only the necessaries. I will try it.
2. Actually, I don't think the data would cause any errors. Another option would be to have a JSON file with the user's data to create, but we should update it when more tests are added. What do you think?

Reply:
1. Thanks!
2. You're right; tests should be idempotent, so it shouldn't matter whether the data changes. We can go ahead and use these for better coverage. We can revisit it if we have issues later on.

Reply:
Done.

Reply:
What @magajh mentioned [here](https://github.com/eduNEXT/eox-core/pull/275#discussion_r1763022408) made me question why we'd need this generator. If test data shouldn't make tests fail and they should be idempotent, why do we need a generator? Can't we manually generate the number of inputs we need, considering edge cases, etc?

Reply:
Yes, we could do that. We can remove the Faker library and include static data for users. What do you think? @magajh @mariajgrimaldi